README.md

```diff
@@ -36,6 +36,56 @@ This will:
 
 Both the creation of model inputs and comparison of model outputs is handled by Claude.
 
+## Example GitHub Actions workflow
+
+Create a new workflow file in `.github/workflows/cog-safe-push.yaml` and add the following:
+
+```yaml
+name: Cog Safe Push
+
+on:
+  workflow_dispatch:
+    inputs:
+      model:
+        description: 'The name of the model to push in the format owner/model-name'
+        type: string
+
+jobs:
+  cog-safe-push:
+    # Tip: Create custom runners in your GitHub organization for faster builds
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.12"
+
+    - name: Install Cog
+      run: |
+        sudo curl -o /usr/local/bin/cog -L "https://github.com/replicate/cog/releases/download/v0.10.0-alpha27/cog_$(uname -s)_$(uname -m)"
```

Comment:
@zeke any reason for using v0.10.0-alpha27?

Reply:
Nope! Just an errant copy-paste from a real-life workflow. Updated to install Cog latest. 👍🏼